sdk/c/api/peer_connection_interface.cc

```diff
@@ -324,6 +324,18 @@ webrtcPeerConnectionInterfaceAddTrack(WebrtcPeerConnectionInterface* connection,
   return cresult;
 }
 
+RTC_EXPORT extern "C" bool
+webrtcPeerConnectionInterfaceAddICECandidate(WebrtcPeerConnectionInterface* connection,
+                                      const char* sdpMid,
```

Comment:
```suggestion
                                      const char* sdp_mid,
```

Follow the style of C++ codes included in the upstream.